frontend/src/utils/DeckMapView.tsx

```diff
@@ -9,6 +9,7 @@ import { Pose } from 'models/Pose'
 import { MapCompass } from 'utils/MapCompass'
 import { Deck } from 'models/Deck'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { getMeta } from 'components/Pages/MissionPage/MapPosition/MissionMapView'
```

Comment:
Should getMeta be in utils instead since its used by both? Maybe in a map utilities file (in a map folder?)?

Reply:
Or maybe not since deckMapView probably will disappear soon